core/ir/opnd_shared.c

```diff
@@ -1321,6 +1367,13 @@ opnd_same(opnd_t op1, opnd_t op2)
     case IMMED_FLOAT_kind:
         /* avoid any fp instrs (xref i#386) */
         return *(int *)(&op1.value.immed_float) == *(int *)(&op2.value.immed_float);
+#ifndef WINDOWS
+        /* Type double currently not included for Windows because sizeof(opnd_t) does
+         * not equal EXPECTED_SIZEOF_OPND, triggering the ASSERT in d_r_arch_init().
+         */
+    case IMMED_DOUBLE_kind:
+        return *(long *)(&op1.value.immed_double) == *(long *)(&op2.value.immed_double);
```

Comment:
`long` is going to have problems as it is 32-bit on Windows and could be elsewhere as well.  Sthg like `int64` would be better.

Reply:
Done.